src/pykiso/test_result/assert_step_report.py

```diff
@@ -276,24 +276,23 @@ def func_wrapper(*args, **kwargs):
             currentframe = inspect.currentframe()
             f_back = currentframe.f_back
             test_name = f_back.f_code.co_name
-            test_class = func.__self__
-            test_class_name = type(test_class).__name__
-            assert_name = func.__name__
+            test_case_inst: TestCase = assert_method.__self__
+            test_class_name = type(test_case_inst).__name__
+            assert_name = assert_method.__name__
 
             # filter parent call, only known function recorded
             parent_method = re.findall(_FUNCTION_TO_APPLY, test_name.lower())
             if parent_method:
                 # Assign variables to signature
-                signature = inspect.signature(func)
+                signature = inspect.signature(assert_method)
                 arguments = signature.bind(*args, **kwargs).arguments
-                test_name = test_class.step_report.current_table or test_name
+                test_name = test_case_inst.step_report.current_table or test_name
 
                 # 1. Gather message, var_name, expected, received
                 # 1.1 Get message. default value: ""
-
-                if test_class.step_report.message:
-                    message = test_class.step_report.message
-                    test_class.step_report.message = ""
+                if test_case_inst.step_report.message:
```

Comment:
Why is codecov complaining in many of the TCs? Can we somehow improve the coverage here?

Reply:
I mostly changed the variable name, so I assume it wasn't tested before either
I'll see what I can do ;)

Reply:
assert_step_report coverage increased by 7%, should be fine now